### PR TITLE
AS-853: Add maintenance window to workspace manager postgres db

### PR DESCRIPTION
**Pull request by ansingh7115**

### CODE REVIEW

**Review by rmeffan — COMMENTED** [review]
See AS-853 for a question about conditional rendering via `cloudsql_maintenance_window_enable`.

File: terra-workspace-manager/cloudsql.tf [hunk 1]
```diff
@@ -26,6 +26,10 @@ module "cloudsql-pg13" {
     record_client_address   = true
   }
 
+  cloudsql_maintenance_window_day = 6
```
Comment (rmeffan) [inline terra-workspace-manager/cloudsql.tf:29]: Are these hard-coded here to use as defaults for the module?

**Review by rmeffan — COMMENTED** [review]
Commenting for the moment; need to ask Chelsea a question when she's out of her current meeting.

ETA: NM; I had something backwards. 

Will thumb when this PR and related PRs have been verified to work with a successful run of `atlantis plan`.

**Review by davidangb — COMMENTED** [review]
once broadinstitute/terraform-shared#163 merges, create a tag in terraform-shared to represent your "release", then update the reference in terra-env/main.tf … otherwise it will be pointing at a nonexistent branch

**Comment by rmeffan** [comment]
Is this change intended to go live for all envs including Terra Prod at the same time?

**Review by davidangb — APPROVED** [review]

**Review by rmeffan — APPROVED** [review]
LGTM.

**Pull request merged**